### PR TITLE
Chennai changes

### PR DESCRIPTION
**Pull request by tomasholderness**
PowerTrack support for Chennai instance (and other cities)
- changed ntwitter module
- removed GRASP dependency (handled by server and cognition-notification-service)
- added different GNIP PowerTrack rules for Chennai

### CODE REVIEW

**Comment by matthewberryman** [comment]
@talltom are you ready for me to test this out for dev.riskmap.in ?

**Comment by tomasholderness** [comment]
@matthewberryman just waiting on @MaanasaPriyaa to fix up some outstanding translation and filtering issues for Chennai. We've got a long way to making this transferable, so we don't have to have separate branches for different regions.

File: PowertrackDataSource.js [hunk 1]
```diff
@@ -504,17 +494,21 @@ PowertrackDataSource.prototype._getCardLink = function(username, network, langua
...
-    method: options.method,
-    headers: options.headers,
-    port: options.port,
+    url: self.config.card_server.address,
+    method: 'POST',
+    headers: {
+        'x-api-key': self.config.card_server.x_api_key,
+        'Content-Type': 'application/json'
+    },
+    port: 80,
```
Comment (matthewberryman) [inline PowertrackDataSource.js]: don't hard code the port in. Make it a variable. @talltom @MaanasaPriyaa

**Comment by matthewberryman** [comment]
@talltom you'd have to get media IDs in place, or make sure Twitter won't barf if that's null, in order to get this deployed. Also are we going to remove the dependency on cognicity-reports and just have this as a standalone thing (more refactoring)?

**Comment by tomasholderness** [comment]
@matthewberryman 
- placeholder media IDs are in cognicity-reports/sample.env, but still waiting for new GIFs for India
- yes I would like to get this is standalone, but lets get India running, and then take a look at this over the summer
- I made port a param and added to configs

**Comment by matthewberryman** [comment]
I've done a bit of work in cognicity-reports and would do more but don't know which twitter module you're planning on using (I tried dropping in twit in uncommitted changes I tried on reports instance) but then had a few issues with that. See https://github.com/urbanriskmap/cognicity-reports/commit/320852793005dd9656580102aa925ed67507468a#commitcomment-22217295 
 for more. I'm out of time today as I need to get slides ready for US trip.

**Pull request merged**